builtin/providers/aws/validators.go

```diff
@@ -1154,3 +1154,19 @@ func validateDbOptionGroupNamePrefix(v interface{}, k string) (ws []string, erro
 	}
 	return
 }
+
+func validateAwsAlbTargetGroupName(v interface{}, k string) (ws []string, errors []error) {
+	name := v.(string)
+	if len(name) > 32 {
+		errors = append(errors, fmt.Errorf("%q (%q) cannot be longer than '32' characters", k, name))
+	}
+	return
+}
+
+func validateAwsAlbTargetGroupNamePrefix(v interface{}, k string) (ws []string, errors []error) {
+	name := v.(string)
+	if len(name) > 32 {
```

Comment:
That the message on the next line says 6! (I came here because I got this error upon trying name_prefix:

> * aws_alb_target_group.mod: Error creating ALB Target Group: ValidationError: Target group name 'ash-test-airflow-webserver0019dc824d8cf5fe93abbb8785' cannot be longer than '32' characters